mara_db/shell.py

```diff
@@ -379,13 +446,20 @@ def __(db: dbs.SnowflakeDB, header: bool = None, footer: bool = None, delimiter_
 
 
 @copy_to_stdout_command.register(dbs.DatabricksDB)
-def __(db: dbs.DatabricksDB, header: bool = None, footer: bool = None, delimiter_char: str = None, csv_format: bool = None):
+def __(db: dbs.DatabricksDB, header: bool = None, footer: bool = None, delimiter_char: str = None, csv_format: bool = None,
+       pipe_format: Format = None):
     assert footer is None, "unimplemented parameter for DatabricksDB"
 
-    if csv_format is None:
-        csv_format = True
+    if pipe_format:
+        if isinstance(pipe_format, CsvFormat):
+            if header is None:
+                header = pipe_format.header
+            if delimiter_char is None:
+                delimiter_char = pipe_format.delimiter_char
+        else:
+            raise ValueError(f'Unsupported pipe_format for DatabricksDB: {pipe_format}')       
 
-    if not csv_format:
+    if csv_format == False:
         raise ValueError('Not supported format: csv_format must be True or not set')
```

Comment:
same here: code is right, comment is wrong?

Reply:
Done